### ISSUE TITLE
tlmgr: unexpected return value from verify_checksum: -5

### ISSUE THREAD

**Issue by VitaminKay** [overleaf/overleaf#747]
## Steps to Reproduce

1. Fresh install of latest Ubuntu 18.04.4/CentOS 7 (tried both)
2. Install Docker Engine and Docker Compose
3. Create and edit docker-compose.yml from [this](https://raw.githubusercontent.com/overleaf/overleaf/master/docker-compose.yml) template in a directory.
4. Execute `docker-compose up`
5. Load web interface to verify initial functionality
6. Attempt to execute `docker exec sharelatex tlmgr install scheme-full` **in another shell window**

## Expected Behaviour

In the past, I would receive an error stating that tlmgr needed to be upgraded. I would execute `docker exec sharelatex tlmgr upgrade --self`, then execute `docker exec sharelatex tlmgr install scheme-full` again and TeXLive packages would then begin installing.

## Observed Behaviour

Now when I execute `docker exec sharelatex tlmgr install scheme-full` I get the following result. I get the same result regardless of whether the base OS is Ubuntu or CentOS.
![annoying sharelatex issue](https://user-images.githubusercontent.com/47240191/83089038-8d809c80-a052-11ea-876d-71438918075f.png)

## Context

This issue is very frustrating and seems to be new. I am unable to use Overleaf like this as my team and I rely on various TeXLive packages that currently cannot be installed. I am trying to install the TeXLive packages as I have done in the past on a new Overleaf instance.
## Technical Info

Operating System(s): Ubuntu 18.04.4/CentOS 7 v2003
Hardware Configuration:
- Hyper-V Virtual Machine (Host is a PowerEdge R930)
- Intel Xeon E7-8867v3 (24x vCPU)
- 8 GB of RAM
- Docker Version: 19.03.9, build 9d988398e7
- Docker Compose Version: 1.25.5, build 8a1c60f6
* Signed in as: `root`

## Analysis

This seems to be an issue related to tlmgr. Is there something wrong with my configuration? This exact setup has worked in the past.

**Comment by VitaminKay** [comment]
I found a way to resolve this! I'm closing the issue.

**This was key information:**
https://tex.stackexchange.com/questions/537376/tlmgr-unexpected-return-value-from-verify-checksum-5-for-main-repository

**My process:**
I used docker exec to get a shell into the running container (`docker exec -it sharelatex /bin/bash`), ran `wget http://mirror.ctan.org/systems/texlive/tlnet/update-tlmgr-latest.sh`, then ran `chmod +x update-tlmgr-latest.sh`, then ran `tlmgr --verify-repo=none install adjustbox`, and finally ran `tlmgr install scheme-full`.

After this series of commands, the TeXLive packages started to install as normal!
